GOVERNANCE.md

```diff
@@ -123,25 +123,24 @@ project, violate the Code Of Conduct, or need to be removed for some
 other reason, they may be removed by a 2/3 majority vote of the other
 Committee members, or a vote of the Steering Committee.
 
-
 #### Subproject Steering Committee Representative
 
 Additionally, the Approver Committee will select, by an election
 conducted by the Operator Framework Election officials, one
 representative of the project to the Operator Framework Steering
 Committee. This representative need not be a member of the project's
-Maintainer Committee, and will be replaced or renewed by the Committee
+Approver Committee, and will be replaced or renewed by the Committee
 annually.
 
 #### Subproject Governance Ammendments
 
-If the project members wish to update or supplement governance
-documents, this will require a ⅔ majority  of project Maintainers
-followed by a ⅔ majority approval from th e Operator Framework steering
-committee.                                
-                                          
-## Steering Committee                     
-                                          
+If a subproject wishes to update or supplement governance
+documents, this will require a 2/3 majority of project Approvers
+followed by a 2/3 majority approval from the Operator Framework steering
+committee.
```

Comment:
This feels like a recipe to make it impossible to ever change governance.  Consider that the #1 reason to change written governance is that the prior rules have become inoperable, usuaully due to contributor departures.  Suggest instead:

```suggestion
If a subproject wishes to update or supplement governance
documents, this will require a 2/3 majority of project Approvers
or 2/3 majority approval from the Operator Framework steering
committee.
```